corsheaders/models.py

```diff
@@ -3,3 +3,7 @@
 
 class CorsModel(models.Model):
     cors = models.CharField(max_length=255)
+
+    @python2_unicode_compatible
+    def __str__(self)
```

Comment:
invalid syntax as there's no `:` at the end of the line, see test failures on travis